### DISCUSSION TITLE
My notification is called twice with `@notify.window=""`

### DISCUSSION THREAD

**Discussion by ghost** [alpinejs/alpine#3829] [7. Bugs (archived)]
Hello everyone, I have a notification blade and I get notifications with `@notify.window=""` but everything is called twice and I can't figure out why for 2 days... I got it with `@persist('notification' )` tried. Now it no longer says +1 on every site switch, but twice, and I didn't find a solution for this because with Javascript, when I use the handler, I got the message once...

![image](https://github.com/alpinejs/alpine/assets/139617589/99164a1d-2e50-4fc4-ac48-760afc47a67d)
![image](https://github.com/alpinejs/alpine/assets/139617589/4baa55c2-6899-41e4-a8dc-e8f65acf5eaa)
![image](https://github.com/alpinejs/alpine/assets/139617589/89342113-45ab-4287-88fd-9afbfe2cafc6)

**Comment by ghost** [comment]
Fixed ... I just found a post and apparently you're not supposed to add the code from the installation at https://alpinejs.dev/essentials/installation and it works now...

![image](https://github.com/alpinejs/alpine/assets/139617589/b42df2f2-e9b2-4c29-9dce-beb8dd688924)

**Reply by ekwoka** [reply]
Yup, this is covered in Livewire 3 upgrade guides
